cli/src/utils.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright 2018 Bitwise IO
```

Comment:
Suggest not creating a utils module dumping ground file. Calling this config or something will give it a better definition and purpose and prevent a place for random ill-defined functions to exist.